internal/server/iigointernal/executive.go

```diff
@@ -105,54 +134,60 @@ func (e *executive) replyAllocationRequest(commonPool shared.Resources, aliveIsl
 			communicateWithIslands(islandID, shared.TeamIDs[e.ID], data)
 		}
 	}
```

Comment:
There is a service charge before the island decides to perform that action. The service charge should only occur if the islands decides to perform the action